rebar.config

```diff
@@ -3,7 +3,7 @@
 {deps, [
         {reltool_util, ".*", {git, "https://github.com/okeuday/reltool_util.git", {tag, "v1.4.0"}}},
         {riak_core, ".*", {git, "git://github.com/Regulators/riak_core", {tag,"2.0.1jb3"}}},
-        {rafter, ".*", {git, "git@github.com:andrewjstone/rafter.git", {branch, "master"}}}
+        {rafter, ".*", {git, "https://github.com/andrewjstone/rafter.git", {tag, "0.2.1"}}}
```

Comment:
I am ok with `https`, but can we stick with the lastest version?

If you uncomfortable to `{branch, "master"}` you can replace it with latest commit id:
`{rafter, ".*", {git, "https://github.com/andrewjstone/rafter.git", "bef9c9b6c5d8cad14685b2f3045dd83d9915e513"}}`